src/main/java/org/broadinstitute/consent/http/db/DAOContainer.java

```diff
@@ -76,6 +77,15 @@ public void setDataAccessRequestDAO(
     this.dataAccessRequestDAO = dataAccessRequestDAO;
   }
 
+  public DarCollectionDAO getDarCollectionDAO() {
+    return darCollectionDAO;
+  }
+
+  public void setDarCollectionDAO(
+    DarCollectionDAO darCollectionDAO) {
+    this.darCollectionDAO = darCollectionDAO;
+  }
```

Comment:
This is only called in a test. It needs to be used in `ConsentModule.providesDAOContainer()` as well so that it has a value. Some of the NPEs I'm seeing are from this not being set.